src/py/run_program.rs

```diff
@@ -53,13 +52,77 @@ pub fn serialize_and_run_program(
     apply_kw: u8,
     max_cost: u32,
     flags: u32,
+) -> PyResult<(u32, Py<PyBytes>)> {
+    let mut opcode_lookup_by_name = HashMap::<String, Vec<u8>>::new();
```

Comment:
I thought the idea that this list would be defined by `clvm`. This is here for backwards compatibility with this API, yeah?
Since this is a global, immutable, constant, is there a way to do this once, rather than every call to `serialize_and_run_program()`?

Reply:
Expediency, with a plan to something less gross later.